common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelInput.java

```diff
@@ -69,10 +69,10 @@ public class MLRegisterModelInput implements ToXContentObject, Writeable {
     public static final String DOES_VERSION_CREATE_MODEL_GROUP = "does_version_create_model_group";
     public static final String GUARDRAILS_FIELD = "guardrails";
 
-    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_DOES_VERSION_CREATE_MODEL_GROUP = Version.V_2_11_0;
-    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_AGENT_FRAMEWORK = Version.V_2_12_0;
-    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_GUARDRAILS_AND_AUTO_DEPLOY = Version.V_2_13_0;
-    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_INTERFACE = Version.V_2_14_0;
+    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_DOES_VERSION_CREATE_MODEL_GROUP = Version.fromString("2.11.0");
+    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_AGENT_FRAMEWORK = Version.fromString("2.12.0");
+    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_GUARDRAILS_AND_AUTO_DEPLOY = Version.fromString("2.13.0");
+    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_INTERFACE = Version.fromString("2.14.0");     
```

Comment:
@dbwiddis Can you help combining those two version variables consuming both 2.14 together to something like `MINIMAL_SUPPORTED_VERSION_FOR_INTERFACE_AND_MODEL_TTL`? I think it could be a better idea to have one version variable for each version, for better maintenance

Reply:
Sure, I'll stick these custom version constants in CommonValue and reference them.

Reply:
That's great.

Reply:
Thanks, @b4sjoo .  Can you or @ylwu-amzn please merge this PR to unblock some related work using it to generate a zip file for use in a 2.13 environment?

Reply:
Merged